README.md

```diff
@@ -1 +1,45 @@
-# bukstore
\ No newline at end of file
+# bukstore
+
+In this project, I have built a simle web app based Bookstore. I've achieved this by using the React.
+
+## Live Link
+
+# Coming soon!
+
+## Install
+
+- [Download](Clone the repository to your machine)
+
+- $ git clone https://github.com/softwareengineermwi/bukstore/tree/dev
+
+- Navigate to the project folder after switching to the 'development' branch and execute the liveServer. You can find this project on your browser where it will be hosted locally.
+
+## Usage
+
+### Development server
+- to run the Webapp you need to use `npm run start`
+
+You can now view the development server at `localhost:8080`.
```

Comment:
### [OPTIONAL]

- [ ] It will be great to actually add the correct hosting port for hosting. In react apps, the default port is 3000. This will make your project even more proffesional.